### ISSUE TITLE
A question about some code in Dataloader.

### ISSUE THREAD

**Issue by swordfate** [dmlc/dgl#4508]
## ❓ Questions and Help
My case is graph topology and features/labels are on CPU, GNN is computed on GPU. UVA is not used.

In Dataloder, when a Thread doing [`prefetcher_entry`](https://github.com/dmlc/dgl/blob/2dd06b21529d77fa398706444f6db080378e8a79/python/dgl/dataloading/dataloader.py#L368) function for each batch (I think it is generated by PyTorch DataLoader), a [`_prefetch`](https://github.com/dmlc/dgl/blob/2dd06b21529d77fa398706444f6db080378e8a79/python/dgl/dataloading/dataloader.py#L382) function is called. The comment says it `fetches node/edge features`. **My question is**, does the following 3 lines of code means collecting necessary features from CPU memory to another place(the feats variable) also in CPU memory? 

If you could give me any hints, I will be very grateful :)

**Comment by yaox12** [comment]
> does the following 3 lines of code ...

Which 3 lines?

**Comment by swordfate** [comment]
> > does the following 3 lines of code ...
> Which 3 lines?

Sorry. I mean the three lines following [here](https://github.com/dmlc/dgl/blob/2dd06b21529d77fa398706444f6db080378e8a79/python/dgl/dataloading/dataloader.py#L332) L. 333-335.

**Comment by yaox12** [comment]
> collecting necessary features from CPU memory to another place(the feats variable) also in CPU memory

No. It will fetch the features to GPU memory in
https://github.com/dmlc/dgl/blob/fce96140896a8580cb26d81a3c24e01c76b7dc28/python/dgl/dataloading/dataloader.py#L259-L260
and finally in 
https://github.com/dmlc/dgl/blob/fce96140896a8580cb26d81a3c24e01c76b7dc28/python/dgl/storages/pytorch_tensor.py#L8-L15
